build.gradle

```diff
@@ -17,6 +17,7 @@ def grpcVersion = "1.39.0"
 def protocVersion = "3.17.3"
 def authzedProtoCommit = "12963402f4474d528d5c14a5162db287"
 def bufDir = "${buildDir}/buf"
+def protocPlatform = project.hasProperty('protoc_platform')
```

Comment:
If we can change this protocPlatformTag and use a ternary operator instead, that'd be far more readable, imo